packages/crawler/tests/e2e/specs/index.e2e.ts

```diff
@@ -0,0 +1,100 @@
+import { setContentWithDesignSystem } from '../helpers';
+import { evaluatePage } from '../../../src/evaluate-page';
+import { Page } from 'puppeteer';
+
+let page: Page;
+beforeEach(async () => (page = await browser.newPage()));
+afterEach(async () => await page.close());
+
+it('should retrieve children and hostPdsComponent correctly', async () => {
+  await setContentWithDesignSystem(page, {
+    bodyHtml: `
+        <p-banner theme="dark">
+          <span slot="title">Some notification title</span>
+          <span slot="description">
+            Some notification description. And some <a href="https://www.porsche.com/">LINK</a> element.
+          </span>
+        </p-banner>
+    `,
+  });
+  console.log('Evaluating page..');
```

Comment:
Let's reduce these logs since they don't bring any value.
<img width="677" alt="Bildschirm­foto 2023-01-17 um 09 09 33" src="https://user-images.githubusercontent.com/4380295/212843195-1b145621-a5f2-4ab7-96a3-3fe828f40034.png">

Reply:
removed these logs